webrtc.html

```diff
@@ -2075,6 +1826,314 @@
+                <var>connection</var>.</p>
+              </li>
+            </ol>
+          </dd>
+
+          <dt>void removeTrack (RTCRtpSender sender)</dt>
+
+          <dd>
+            <p>Removes <var>sender</var>, and its associated <code>MediaStreamTrack</code>, from the
+            <code><a>RTCPeerConnection</a></code>.</p>
+
+            <!-- onended -->
+            <p>When the other peer stops sending a track in this manner, a
+            <code title="event-MediaStreamTrack-ended"><a href=
+            "#event-mediastream-ended">ended</a></code> event is
+            fired at the <code><a>MediaStreamTrack</a></code> object.</p>
+
+            <p>When the <dfn id="dom-peerconnection-removetrack"><code title=
+            "">removeTrack()</code></dfn> method is invoked, the user agent
+            MUST run the following steps:</p>
+
+            <ol>
+              <li>
+                <p>Let <var>connection</var> be the
+                <code><a>RTCPeerConnection</a></code> object on which the
+                <code><a>RTCRtpSender</a></code>, <var>sender</var>, is to be
+                removed.</p>
+              </li>
+
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>closed</code>, throw an
+                <code>InvalidStateError</code> exception.</p>
+              </li>
+
+              <li>
+                <p>If <var>sender</var> is not in <var>connection</var>'s
+                <a href="#senders-set">set of senders</a>, then abort
+                these steps.</p>
+              </li>
+
+              <li>
+                <p>Remove <var>sender</var> from <var>connection</var>'s
+                <a href="#senders-set">set of senders</a>.</p>
+              </li>
+
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>stable</code>, then fire a <a href=
+                "#event-negotiation">negotiationneeded</a> event at
+                <var>connection</var>.</p>
+              </li>
+            </ol>
+          </dd>
+
+          <dt>attribute EventHandler ontrack</dt>
+
+          <dd>This event handler, of event handler event type <code><a href=
+          "#event-track">track</a></code>, MUST be fired
+          by all objects implementing the <code><a>RTCPeerConnection</a></code>
+          interface. It is called any time a <code>MediaStreamTrack</code> is added
+          by the remote peer. This will be fired only as a result of
+          <code>setRemoteDescription</code>. ontrack happens as early as
+          possible after the <code>setRemoteDescription</code>. This callback
+          does not wait for a given media track to be accepted or rejected via
+          SDP negotiation.</dd>
```

Comment:
This might need a bit more detail explaining the difference for PRans and ans. We also need to describe what happens  to a track that arrived as early media but was rejected in the Ans. (it should be terminated)